10-2.4/Dockerfile

```diff
@@ -15,3 +15,6 @@ RUN mkdir -p /docker-entrypoint-initdb.d
 COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/postgis.sh
 COPY ./update-postgis.sh /usr/local/bin
 
+COPY ./healthcheck.sh /usr/local/bin/docker-healthcheck
+RUN chmod +x /usr/local/bin/docker-healthcheck
+HEALTHCHECK CMD ["docker-healthcheck"]
```

Comment:
Let's remove this line and add a section to the `README.md` telling users how to enable the health check command using either `HEALTHCHECK CMD` or the `--health-cmd` command line switch (I think it would make sense to link to https://docs.docker.com/engine/reference/run/#healthcheck).

The `README.md` section should describe what `docker-healthcheck` does and perhaps possible reasons why someone would want a more specific healthcheck for their application, such as the fact that it can't test connectivity from outside the container.